Resources/doc/cookbook/recipe_custom_action.rst

```diff
@@ -92,7 +92,10 @@ to implement a ``clone`` action.
 
     class CRUDController extends Controller
     {
-        public function cloneAction()
+        /**
+         * @param $id
+         */
+        public function cloneAction($id)
```

Comment:
Do you mean a method to get the id, directly as the second sprintf() argument, in place of the variable?